apps/api/src/app/inbound-parse/inbound-parse.module.ts

```diff
@@ -1,22 +1,21 @@
 import { MiddlewareConsumer, Module, NestModule } from '@nestjs/common';
-import { CompileTemplate, QueuesModule } from '@novu/application-generic';
+import { BaseAppQueuesModule, CompileTemplate } from '@novu/application-generic';
 
 import { USE_CASES } from './usecases';
 import { InboundParseController } from './inbound-parse.controller';
 import { InboundParseQueueService } from './services/inbound-parse.queue.service';
 import { GetMxRecord } from './usecases/get-mx-record/get-mx-record.usecase';
-import { InboundEmailParse } from './usecases/inbound-email-parse/inbound-email-parse.usecase';
 
 import { SharedModule } from '../shared/shared.module';
 import { AuthModule } from '../auth/auth.module';
 
 const PROVIDERS = [InboundParseQueueService, GetMxRecord, CompileTemplate];
 
 @Module({
-  imports: [SharedModule, AuthModule, QueuesModule],
+  imports: [SharedModule, AuthModule, BaseAppQueuesModule],
   controllers: [InboundParseController],
   providers: [...PROVIDERS, ...USE_CASES],
-  exports: [...USE_CASES, QueuesModule],
```

Comment:
Do we need to export this one here?

Reply:
It probably not anymore. I think it can be safe to be removed.